packages/addons/vitest-addon/index.ts

```diff
@@ -38,63 +38,143 @@ export default defineAddon({
-					importDecl.importKind === 'value' &&
-					importDecl.specifiers?.some(
-						(specifier) =>
-							specifier.type === 'ImportSpecifier' && specifier.imported.name === 'defineConfig'
-					)
-			);
-
-			// we'll need to replace the "vite" import for a "vitest/config" import.
-			// if `defineConfig` is the only specifier in that "vite" import, remove the entire import declaration
-			if (defineConfigImportDecl?.specifiers?.length === 1) {
-				const idxToRemove = ast.body.indexOf(defineConfigImportDecl);
-				ast.body.splice(idxToRemove, 1);
-			} else {
-				// otherwise, just remove the `defineConfig` specifier
-				const idxToRemove = defineConfigImportDecl?.specifiers?.findIndex(
-					(s) => s.type === 'ImportSpecifier' && s.imported.name === 'defineConfig'
-				);
-				if (idxToRemove) defineConfigImportDecl?.specifiers?.splice(idxToRemove, 1);
-			}
+		if (kit) {
+			sv.devDependency('@testing-library/svelte', '^5.2.4');
+			sv.devDependency('@testing-library/jest-dom', '^6.6.3');
+			sv.devDependency('jsdom', '^25.0.1');
+
+			sv.file(`${kit.routesDirectory}/page.svelte.test.${ext}`, (content) => {
+				if (content) return content;
 
-			const config = common.expressionFromString('defineConfig({})');
-			const defaultExport = exports.defaultExport(ast, config);
+				return dedent`
+					import { describe,test, expect } from 'vitest';
+					import { render, screen } from '@testing-library/svelte';
+					import Page from  './+page.svelte';
 
-			const test = object.create({
-				include: common.expressionFromString("['src/**/*.{test,spec}.{js,ts}']")
+					describe('/+page.svelte',()=>{
+						test('should render h1',()=>{
+							render(Page);
+							expect(screen.getByRole('heading',{level:1})).toBeInTheDocument();
+						})
+					})
+				`;
 			});
 
-			// uses the `defineConfig` helper
-			if (
-				defaultExport.value.type === 'CallExpression' &&
-				defaultExport.value.arguments[0]?.type === 'ObjectExpression'
-			) {
-				// if the previous `defineConfig` was aliased, reuse the alias for the "vitest/config" import
-				const importSpecifier = defineConfigImportDecl?.specifiers?.find(
-					(sp) => sp.type === 'ImportSpecifier' && sp.imported.name === 'defineConfig'
+			sv.file('vitest-setup-client.ts', (content) => {
+				if (content) return content;
+
+				return dedent`
+					import '@testing-library/jest-dom/vitest'
+
+					// add global mocks here, i.e. for sveltekit '$app/stores'
+				`;
+			});
+
+			sv.file('vitest.workspace.ts', (content) => {
+				const { ast, generateCode } = parseScript(content);
+
+				imports.addNamed(ast, 'vitest/config', { defineWorkspace: 'defineWorkspace' });
+				imports.addNamed(ast, '@testing-library/svelte/vite', { svelteTesting: 'svelteTesting' });
+
+				const clientObjectExpression = object.create({
+					extends: common.createLiteral(`./vite.config.${ext}`),
+					plugins: common.expressionFromString(
+						'[svelteTesting({ resolveBrowser: true, autoCleanup: true })]'
```

Comment:
these 2 might be defaults so we could just use `svelteTesting()` maybe ping someone from testing library svelte here?